docs/reference/content/driver/tutorials/databases-collections.md

```diff
@@ -200,4 +200,12 @@ MongoDatabase database = client.getDatabase("mydb")
 // or per collection
 MongoCollection<Document> collection = database.getCollection("mycoll")
                                                .withCodecRegistry(codecRegistry);
-```
+```   
+
+{{% note %}}
+Starting with the 3.12 release of the driver, you can also change the encoding of `UUID` instances via the `uuidRepresentation` property of
+`MongoClientSettings`.  See 
+[`MongoClientSettings.getUuidRepresentation`]({{<apiref "com/mongodb/MongoClientSettings.html#getUuidRepresentation()">}}) for
```

Comment:
I think this link is correct, but if so, all the others are wrong on this branch.  Perhaps we should backport the changes from the 4.x branch?

Reply:
Sigh! It depends on the version of javadoc. Can we publish just using java 8 for 3.12?

Reply:
I hate to do it since the Java 11 docs look so much nicer.   Did you make the updates of the javadoc links in the 4.x branch in a separate commit?  If so, I could try to backport them.

Reply:
Looks like it's not a separate commit so we can't just cherrypick it.

But I think it's worth it to invest an hour in manually backporting the updates to the links.  I'll open a ticket for that.